xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -686,11 +707,23 @@ void CPVRGUIInfo::CharInfoPlayingClientName(std::string &strValue) const
 
 void CPVRGUIInfo::CharInfoEncryption(std::string &strValue) const
 {
-  CPVRChannelPtr channel(CServiceBroker::GetPVRManager().Clients()->GetPlayingChannel());
-  if (channel)
-    strValue = channel->EncryptionName();
+  if (m_descrambleInfo.iCaid != -1)
```

Comment:
<code>const int PVR_DESCRAMBLE_INFO_UNKNOWN_VALUE = 1;</code> maybe? Is generic enough to fit for caid, hops, ecmtime etc